new_tests/test_xmltompc80col.py

```diff
@@ -68,18 +68,16 @@ def test_provid_notrksub():
     )
     assert os.path.exists(outfile) and os.stat(outfile).st_size != 0
     
-    
-@pytest.mark.xfail
 def test_trksub_9char():
     """ The code should stop with an error for any trksubs longer than 8 chars"""
     infile = "input/trksub_9chars.xml"
     outfile = "output/trksub_9chars.obs"
     # Remove outfile if there
     if os.path.exists(outfile):
         os.remove(outfile)
-    subprocess.run(
+    result = subprocess.run(
         "python3 ../Python/bin/xmltompc80col.py " + infile + " " + outfile,
         shell=True,
         check=False,
     )
-    assert os.path.exists(outfile) and os.stat(outfile).st_size != 0
\ No newline at end of file
+    assert result.returncode != 0 # Designed to fail...
```

Comment:
So this test was not working right because the (expected) failure mode created a non-empty file. So it returned XPASS for every size of trkSub.

After much testing of dead ends, I think this now does what is intended, but I suppose there are better (and more specific) ways to do the test...